src/api-service/__app__/onefuzzlib/azure/creds.py

```diff
@@ -81,6 +81,14 @@ def get_instance_url() -> str:
     return "https://%s.azurewebsites.net" % get_instance_name()
 
 
+@cached
+def get_agent_instance_url() -> str:
+    if os.environ.get("ONEFUZZ_USE_NET_AGENT_FUNCTIONS") == "true":
```

Comment:
this should probably be `ONEFUZZ_USE_DOTNET_AGENT_FUNCTIONS`

Reply:
we could actually have this automatically enabled if the Python functions are disabled, since that is done by environment variable